app/api_routes.py

```diff
@@ -309,7 +309,7 @@ def api_chatbot(chatbot_id: int) -> Union[Response, tuple[Response, int]]:
     query: str = data.get("query")
     apikey = request.headers["apikey"]
     engine = request.headers["engine"]
-    chat_to_pass: List[Dict[str, str]] = [{"role": "system", "content": chatbot.prompt}]
+    chat_to_pass: List[Dict[str, str]] = [{"role": "system", "content": query}]
```

Comment:
We need this

```suggestion
    chat_to_pass: List[Dict[str, str]] = [{"role": "system", "content": chatbot.latest_version.prompt}]
```

Reply:
@priyanshuverma-dev : thanks for suggestion